docs/source/contributing.rst

```diff
@@ -0,0 +1,154 @@
+**********************************
+Contributing to This Documentation
+**********************************
+
+.. highlight:: none
+
+Getting Started
+==========================
+
+This documentation is created using 
+`Sphinx <http://www.sphinx-doc.org/en/stable>`_. Sphinx is an open-source tool 
+that makes it easy to create intelligent and beautiful documentation, written 
+by Georg Brandl and licensed under the BSD license.
+
+The documentation is maintained in the ``master`` branch of the GitHub repository.
+You can include code and its corresponding documentation updates in a single pull-request (PR).
```

Comment:
I don't usually hyphenate `pull-request`.

Reply:
Good catch, I'll remove the hyphen